server.js

```diff
@@ -14,10 +16,51 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
+const Nomination = mongoose.model("Nomination", {
+  "year_film": Number,
+    "year_award": Number,
+    "ceremony": Number,
+    "category": String,
+    "nominee": String,
+    "win": Boolean
+
+}); 
+
+// reseting database is to educational purposes, to not have duplicates
+// in production we should not use it without special order from menagers
+if (process.env.RESET_DB) {
+  const resetDataBase = async () => {
+      await Nomination.deleteMany(); // waiting to delete all data before we create a new object
+    const addedNominate = new Nomination({
+      year_film: 2044, 
+      year_award: 2045,
+      ceremony: 165,
+      category: "best tragic comedy",
+      nominee: "smurf",
+      win: false
+      
+    }); 
+    
+    const addedNominate2 = new Nomination({
+      _id: ObjectId,
+      year_film: 2023, 
+      year_award: 2015,
+      ceremony: 65,
+      category: "best actor",
+      nominee: "Johny Deep",
+      win: true
+    
+    });
+    addedNominate.save();// we need to put it in the reset function because if is after the deleteMany happens before and we do not update the DB
+    addedNominate2.save()
+  }
+  resetDataBase();
+}
```

Comment:
right now you only have two data objects in your api, and to add more from the golden globe seperately would be too much :) a suggestion to get more data in to your DB at the same time from the json file is you can write a forEach iteration:
```

if(process.env.RESET_DB) {
  const resetDataBase = async () => {
    await Nomination.deleteMany();
    goldenGlobesData.forEach(singleNomination => {
      const newNomination = new Nomination(singleNomination);
      newNomination.save();
    })
  }
  resetDataBase();
}
```